client/cypress/end-to-end/ballot-comparison-audit-creation.spec.js

```diff
@@ -0,0 +1,370 @@
+import 'cypress-file-upload'
+
+before(() => cy.exec('./cypress/seed-test-db.sh'))
+
+describe('Ballot Comparison Audit Creation', () => {
```

Comment:
It seems like the audit board test suite also has to do the entire audit creation and setup, which covers the same functionality as this test suite. I would recommend combining them into one test suite that goes through the entire audit flow. It will be faster to run and less test code to maintain.

Reply:
It's also a bit confusing to me that there is state maintained between the different test cases within a test suite. That's generally not a recommended pattern because the test cases can't be run separately, and it seems like Cypress docs also advise against it: https://docs.cypress.io/guides/references/best-practices.html#Having-tests-rely-on-the-state-of-previous-tests

Reply:
Yeah, I think I'm going to restructure things down to just a couple suites with happy paths and get those merged, then figure out the most efficient way to cover the necessary error conditions without any bloat.

Could you elaborate on what you mean by maintained state? Do you mean the id? I haven't noticed any tests that break when run independently and I've run almost all if not all with `.only` during testing.

Reply:
Sounds good :)

By maintain state, I meant state in the database. But I think I just misread the tests because there's a `beforeEach` happening to set up the audit before each `it` test case. My bad.

I guess the root of that point though is that if the `beforeEach` has a lot of steps, running it before each `it` will make the tests take a lot longer, so making one long `it` for each audit you want to run might save a lot of time.